hedera-spring/src/test/java/com/openelements/hedera/spring/test/NftRepositoryTests.java

```diff
@@ -146,7 +146,8 @@ void findByAccountId() throws Exception {
         hederaTestUtils.waitForMirrorNodeRecords();
 
         //when
-        final List<Nft> result = nftRepository.findByOwner(newOwner);
+        final Page<Nft> slice = nftRepository.findByOwner(newOwner);
```

Comment:
I updated the tests for the current methods that support pagination and added some tests that checks a return count that is bigger than the size of 1 page. By doing so we can directly check if the pagination works. Can you please add such tests, too.

Reply:
Tests added. Took me while to figure it out.